website/public/js/controllers/rootCtrl.js

```diff
@@ -215,7 +215,18 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
 
       var gems = User.user.balance * 4;
 
-      var string = (type == 'weapon') ? window.env.t('weapon') : (type == 'armor') ? window.env.t('armor') : (type == 'head') ? window.env.t('headgear') : (type == 'shield') ? window.env.t('offhand') : (type == 'back') ? window.env.t('back') : (type == 'body') ? window.env.t('body') : (type == 'headAccessory') ? window.env.t('headAccessory') : (type == 'eyewear') ? window.env.t('eyewear') : (type == 'hatchingPotions') ? window.env.t('hatchingPotion') : (type == 'eggs') ? window.env.t('eggSingular') : (type == 'quests') ? window.env.t('quest') : (item.key == 'Saddle') ? window.env.t('foodSaddleText').toLowerCase() : type; // FIXME this is ugly but temporary, once the purchase modal is done this will be removed
+      var string = (type in equipmentList) ? equipmentList.get(type, window) : type;
+      // or
+      var string = (type in equipmentList) ? (equipmentList.type.needsLowerCase) ? equipmentList.get(type, window).toLowerCase() : equipmentList.get(type, window) : type;
+      // or
+      var string;
```

Comment:
Just wondering why you'd like to make a local reference to Content.gemPurchasableItems?